vector-im/element-android
# [Device Management] Render extended device info (PSG-773)

## Type of change


## Content


Adding Application section in the session details screen.
Adding extended device info in the Device section of the session details screen depending on the device type.

## Motivation and context


Closes #7294 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/194316797-5e09ef21-1a0f-4a5a-a0e8-23f44f7eb004.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/194316826-7da8a68d-7c9c-4fbf-b96e-4575abc1d401.png" width=300 />
<img src="https://user-images.githubusercontent.com/46314705/194316860-905eca63-f4a1-42d4-b23a-c2e6a6bba2ce.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/194316887-96d4d578-2c2a-4cc5-bfe9-8859de54b757.png" width=300 />

## Tests



- Login to an account in https://synapse-prefill.lab.element.dev/ (Synapse deployment with experimental feature flags for MSC3852 (exposure of user agent))
- Enable the new device manager feature flag
- Go to Settings -> Security & Privacy -> Show all sessions (WIP, V2)
- Go to the session details screen of session with different types
- Check the information is correct

## Tested devices

- OS version(s): Android 11

## Checklist
